proofs/src/circuit/floor_planner/single_pass.rs

```diff
@@ -373,7 +373,8 @@ impl<'a, F: Field, CS: Assignment<F> + 'a + SyncDeps> RegionLayouter<F>
 
 #[cfg(test)]
 mod tests {
-    use midnight_curves::pasta::vesta;
+    use ff::Field;
+    use midnight_curves::Fq;
```

Comment:
Having support for several curves, we should expose fields with explicit mention of the curve.